### PR TITLE
DOCS-3949 - CSM

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?

- Documents CSM Overview
- Restructures Security section nav
- Adds CSM category to Rules filters

### Motivation
DOCS-3949

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/cloud_security_management/_index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    tag: "Blog"
+    text: "Elevate AWS threat detection with Stratus Red Team"
+  - link: "https://www.datadoghq.com/blog/kubernetes-security-best-practices/"
+    tag: "Blog"
+    text: "Best practices for securing Kubernetes applications"
+---
+
+## Overview
+
+Datadog Cloud Security Management delivers real time threat detection and continuous configuration audits across your entire cloud infrastructure - in a unified view for seamless collaboration and faster remediation.  Powered by observability data, security teams can determine the impact of a threat quickly by tracing the full attack flow and easily identify the resource owner where a vulnerability was triggered.  Engineers are equipped to actively monitor their security risks by incorporating security metrics into their existing workflow.
```
[the remark this answers is not in the document]
Comment (hdelaby) [reply content/en/security_platform/cloud_security_management/_index.md:24]: I would say in general, if I compare this with the intro of APM or Log Management in the doc, this feels very weak. It's unclear what capabilities I can expect from the product, until I read the description of CSPM and CWS separately.

**Review by hdelaby — COMMENTED** [review]
Hope I did not have a glance too soon 🤓

File: content/en/security_platform/cloud_security_management/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (n06) [reply content/en/security_platform/cloud_security_management/_index.md:24]: @hdelaby this is copy we took straight from PMM and the investor day presentations, it is not new. I wanted to keep the description consistent with our public announcements

File: content/en/security_platform/cloud_security_management/_index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Cloud Security Management includes [Cloud Security Posture Management (CSPM)](#cloud-security-posture-management), and [Cloud Workload Security (CWS)](#cloud-workload-security).
+
+{{< img src="security_platform/csm_overview.png" alt="Cloud Security Management in Datadog" width="100%">}}
```
[the remark this answers is not in the document]
Comment (n06) [reply content/en/security_platform/cloud_security_management/_index.md:28]: Yes! @kayayarai here is a new screenshot: https://a.cl.ly/P8u72dml

[the remark this answers is not in the document]
Comment (n06) [reply content/en/security_platform/cloud_security_management/_index.md:28]: Scratch that, sorry @kayayarai lets use this one instead please: https://a.cl.ly/P8u72d56

[the remark this answers is not in the document]
Comment (n06) [reply content/en/security_platform/cloud_security_management/_index.md:28]: Yes, here it is: https://a.cl.ly/7KukzJEE

File: content/en/security_platform/cloud_security_management/_index.md (hunk 1 above)
Comment (jtappa) [inline content/en/security_platform/cloud_security_management/_index.md:24]:
```suggestion
Datadog Cloud Security Management delivers real-time threat detection and continuous configuration audits across your entire cloud infrastructure - in a unified view for seamless collaboration and faster remediation.  Powered by observability data, security teams can determine the impact of a threat quickly by tracing the full attack flow and identify the resource owner where a vulnerability was triggered. Engineers are able to actively monitor their security risks by incorporating security metrics into their existing workflow.
```

File: content/en/security_platform/cloud_security_management/_index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+{{< img src="security_platform/cspm_overview.png" alt="Cloud Security Posture Management scores in Datadog" width="100%">}}
+
+## Cloud Workload Security
+
+[Cloud Workload Security (CWS)][2] monitors file and process activity across your environment to detect threats to your infrastructure, like AWS EC2 instances, and workloads, like Kubernetes clusters, in real time at the kernel level. Cloud Workload Security uses the unified Datadog Agent, so if you're already using Datadog to monitor your environment, there's no need to provision additional resources.
+
+{{< img src="security_platform/cws_overview.png" alt="Cloud Workload Security coverage views in Datadog" width="100%">}}
+
+To get started with Datadog Security, navigate to the [Setup & Configuration][3] section in the Datadog app, which has detailed information for single or multi-configuration, or follow the getting started sections below to learn more about each area of the platform.
```
Comment (jtappa) [inline content/en/security_platform/cloud_security_management/_index.md:43]: This is the end of the file, there's no getting started sections to see below...?
```suggestion
To get started with Datadog Security, navigate to the [Setup & Configuration][3] section in Datadog, which has detailed information for single or multi-configuration, or follow the getting started sections to learn more about each area of the platform.
```

**Review by jtappa — CHANGES_REQUESTED** [review]

**Review by jtappa — APPROVED** [review]

**Pull request merged**